suites/suites_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestSuites_Find(t *testing.T) {
```

Comment:
it's useful to have a prefix for the tests so that you can:
```
go test -run ^TestSuites
```
If one day there are multiple files in the package

Reply:
Just thought it was cleaner... my instinct is always the smallest identifier that is unique. But I get your point and fixed it.